chandra_aca/dark_subtract.py

```diff
@@ -0,0 +1,431 @@
+from functools import lru_cache
+
+import numba
+import numpy as np
+import requests.exceptions
+from astropy.table import Table, vstack
+from cheta import fetch_sci
+from cxotime import CxoTime, CxoTimeLike
+from mica.archive import aca_l0
+from mica.archive.aca_dark import get_dark_cal_props
+from ska_helpers import retry
+
+from chandra_aca import maude_decom
+from chandra_aca.dark_model import dark_temp_scale_img
+
+
+def get_dark_data(time: CxoTimeLike):
+    """
+    Retrieve the dark calibration image and temperature for a given time.
+
+    This is a light wrapper around mica.archive.aca_dark.get_dark_cal_props.
+
+    Parameters
+    ----------
+    time : CxoTimeLike
+        Reference time for the dark calibration image.
+
+    Returns
+    -------
+    dc_img : np.array
+        Dark calibration image.
+    dc_tccd : float
+        Dark calibration CCD temperature.
+    """
+    dc = get_dark_cal_props(time, "nearest", include_image=True, aca_image=False)
+    dc_img = dc["image"]
+    if "t_ccd" in dc:
+        dc_tccd = dc["t_ccd"]
+    elif "ccd_temp" in dc:
```

Comment:
It looks like old data doesn't have 't_ccd' set, but modern data has both 't_ccd' and 'ccd_temp' set, so this logic is unnecessary.